design-collaboration-guidelines.md

```diff
@@ -0,0 +1,82 @@
+# Design Collaboration Guidelines
+
+As an app developer to provide the best version of the design, you need to define a standard vocabulary between the design team and you.
+
+Designers are more and more integrating design systems as part of their work, and we can benefit from that to layout common ground. Here below we would define the minimum standard requirements to keep consistency and the usuals forgotten piece of app design. Independently of the size of your team, there is already a set of best practices applied by your designer that you can benefit, and other you will need anyway at some point during the development of the app.
+
+* [A COMPREHENSIVE GUIDE TO DESIGN SYSTEMS](https://www.invisionapp.com/blog/guide-to-design-systems/)
+
+## Style guide - Foundation
+These are the basic style that will serve as the foundation of all other components. And avoid mistakes by not using the proper element. Independently if you follow material design, fluent design or any other guidelines, these basics style element will facilitate design adjustment and rebranding.
```

Comment:
``These are the basic style ``~> ``These are the basic styles``?  ``These are the basic style components``?

Reply:
Could we add an example of the format or where the style guide will be placed?

Reply:
What do you mean by `where the style guide will be placed`?

Reply:
I was wondering if we could explain how a design team can generate a style guide a developer can review when needed and where this content is going to be placed. Is going to be a private URL, a public PDF or any other format we could save on google drive?

Reply:
Personally, for now, I don't really care about that. It could be a lot of possibilities Invision, zeplin or a PDF or whatever the design team chooses. And it's more a general tooling thing, we could mention it in tooling section.